docs/how_to/install_puppet_enterprise.md

```diff
@@ -0,0 +1,107 @@
+# How-To Install Puppet Enterprise
+
+Welcome to the choose your own adventure of installing Puppet Enterprise (PE)!
+This doc will guide you through the process beaker's DSL helpers use to install
+PE for acceptance testing.
+
+Note that this is not a complete documentation of the process, but a general
+overview. There will be specific hiccups for particular platforms and special
+cases. These are not included at this point. The idea is that as we come upon
+them, we will now have a place to document those details, so that we can
+over time bring this to 100% completeness.
+
+# First Things First: What Do You Want to Install?
```

Comment:
Hmm, this only makes me feel stronger that we should have a specific doc on `install_puppet_agent_pe_promoted_repo_on` and link to that doc from here; I don't like the idea of conflating these two different things in a single document.